sideci_demo/rubocop_demo.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# GitHub Repository
+class Repository
+  attr_reader :client
+
+  def initialize(owner, name)
+    @owner = owner
+    @name = name
+    @client = Github.new
+  end
+
+  def releases
+    repository&.releases.map(&:name)
+  end
+
+  def active_branch
+    repository.branches.max { |a, b| a.updated_at <=> b.updated_at }
```

Comment:
> Performance/CompareWithBlock: Use `max_by(&:updated_at)` instead of `max { |a, b| a.updated_at <=> b.updated_at }`.
> [link](https://sideci.com/gh/109700694/pull_requests/1)

問題なし